SerpentsHand/Plugin.cs

```diff
@@ -74,95 +74,19 @@ public override void Register()
 			AddCommands(new string[] { "spawnshsquad" }, new SpawnSquad());
 		}
 
-		public static int LevenshteinDistance(string s, string t)
+		public static Player FindPlayer(string identifier)
```

Comment:
Levenshtein outdated and unnessesary, GetPlayers filter does the same thing in later Smod iterations